doc/ref_impl/cntt-ri/chapters/chapter06.md

```diff
@@ -61,241 +63,400 @@ Thanks to the descriptor file, the NFVi infrastructure deployment could be compl
+| LOCAL_NAME | String | Yes | local device name, LOCAL_NAME must reference either "Network Device Name" from table 6.3.5 |
+| LOCAL_TYPE | String | Yes | Local device type, switch or server |
+| LOCAL_PORT | String | Yes | connected port in local device |
+| REMOTE_RACK | String | Yes |  |
+| REMOTE_NAME | String | Yes | connected remote device name, REMOTE_NAME must reference either "Network Device Name" from table 6.3.5 or "Server Name" from table 6.3.4.1 |
+| REMOTE_TYPE | String | Yes | remote device type, it can be switch or server |
+| REMOTE_PORT | String | Yes | connected port in remote device. When describing port for remote servers, we use port number like 1_1, or 1_2, instead of PCIeslot number, because the server NIC mapping is already defined in 6.3.4.2 |
+| LINE_TYPE | String | Yes | Line type to describe local device type and remote device type, how each line is connected. For example "S-SRV-C_S-TOR" means this line is connecting a service server in compute module to service TOR, and another example "ST-SRV-S_M-TOR" means storage server connecting to a manage TOR in storage module. The line type can be customized defined, as long as it's unified in end user.|
+
+<p align="center"><b>Table 6-3-6:</b> Port mapping information.</p>
 
-<p align="center"><b>Table 6-3-6:</b> Controller information.</p>
 
 <a name="6.3.7"></a>
-### 6.3.7 Compute information
-List of compute nodes that designed for current VIM deployment.
+### 6.3.7 Network planning information
+Network planning info for resource pool needs to be defined, which should include Vlan ID, allocated IP range, the applied node set.
+
+Multiple instances are expected, one instance for each network plane.
 
 | Field # | type | mandatory | Instruction |
 |----|--------------------|----------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| device_name | String | Yes | must be the device_name defined in server_information(###6.3.3), which would be set on Server from BMC|
-| node_name | String | Yes | node name for the compute to deploy, e.g: compute01 |
-| az_name | String | Yes | availability zone name which this node belongs to |
-| ha_name | List | Yes | HA name(s) which this node assigned to.|
-| vnic_type | String | Yes | VIM verification needs know compute node type: OVS,DPDK_OVS or SR-IOV node.|
-| os_reserved_cores | String | No | Designed and planned by provider itself, may be none|
-| ovs_reserved_cores | String | No | Designed and planned by provider itself, may be none |
+| APPLICATION_LAYER | String | Yes |  VIM  or storage |
+| DOMAIN | String | Yes | name of VIM/storage software product  |
+| VENDOR_NETWORK_PLANE | String | Yes | network plane designed/needed by software product |
+| NETWORK_PLANE | String | Yes | corresponding network plane in user view, like Manage, Storage, service  |
```

Comment:
"service" = user plane?

Reply:
How about Network domain?
this table tends  to describe network configuration requirements for Manage/Storage/Service (different network domain)

Reply:
ok